README.md

```diff
@@ -1,44 +1,60 @@
 
 * * *
 
-[![Circle CI](https://circleci.com/gh/drn/jira-cli.svg?style=svg)](https://circleci.com/gh/drn/jira-cli)
-[![Dependency Status](https://gemnasium.com/darrenli/jira-cli.png)](https://gemnasium.com/darrenli/jira-cli)
-[![Code Climate](https://codeclimate.com/github/darrenli/jira-cli.png)](https://codeclimate.com/github/darrenli/jira-cli)
+[![Dependency Status](https://gemnasium.com/SDarayan/jira-cli.png)](https://gemnasium.com/SDarayan/jira-cli)
 
 ### Available Commands
 
     jira all             # Describes all local branches that match JIRA ticketing syntax
+    jira assign          # Assign a ticket to a user
     jira attachments     # View ticket attachments
-    jira browse          # Opens the current input ticket in your browser
     jira comment         # Add a comment to the input ticket
+    jira commentd        # Delete a comment to the input ticket
     jira comments        # Lists the comments of the input ticket
+    jira commentu        # Update a comment to the input ticket
     jira commit          # Commits uncommitted work with the ticket name and summary.
-    jira clipboard       # Copies the url of the JIRA ticket into the clipboard
+    jira delete          # Deletes a ticket in JIRA and the git branch
     jira describe        # Describes the input ticket
     jira install         # Guides the user through JIRA installation
     jira log             # Logs work against the input ticket
+    jira logd            # Deletes work against the input ticket
+    jira logs            # Lists work against the input ticket
+    jira logu            # Updates work against the input ticket
     jira new             # Creates a new ticket in JIRA and checks out the git branch
     jira rename          # Updates the summary of the input ticket
-    jira sprint          # Lists all tickets in active sprint
+    jira tickets         # List the tickets of the input username
     jira transition      # Transitions the input ticket to the next state
+    jira unvote          # Unvote against the input ticket
+    jira unwatch         # Unwatch against the input ticket
     jira version         # Displays the version
+    jira vote            # Vote against the input ticket
+    jira votes           # List the votes of the input ticket
+    jira watch           # Watch against the input ticket
+    jira watchers        # List the watchers of the input ticket
     jira help [COMMAND]  # Describe available commands or one specific command
 
 ### Gem Installation
 
 Rubygems:
 
-    gem install jira-cli
+    gem install jira-cli2
 
 Manually:
 
-    git clone git@github.com:darrenli/jira-cli.git
+    git clone git@github.com:sdarayan/jira-cli.git
     cd jira-cli
     ./scripts/install
 
+Gemfile:
+
+    source 'https://rubygems.org'
+
+    gem 'jira-cli2', '~> 0.2.0', '>= 0.2.0'
+    gem 'inquirer', '~> 0.2.1', '>= 0.2.1', :git => 'https://github.com/arlimus/inquirer.rb.git', :ref => '239cff5afe562870f2f3e086ce78b178bdad319c'
```

Comment:
This explicit inquirer reference shouldn't be necessary as it is part of the gemspec.
Also, this gem is meant to be a CLI exclusively and is not meant to be included in other project. It doesn't really make sense to be part of any gemfile unless the gemfile is used for setting up your development environment rather than a specific project.